.github/workflows/merge-kube-state-metrics.yaml

```diff
@@ -13,7 +13,10 @@ jobs:
       downstream: openshift/kube-state-metrics
       sandbox: rhobs/kube-state-metrics
       go-version: 1.17
-      restore-upstream: CHANGELOG.md VERSION
+      restore-upstream: >-
+        CHANGELOG.md 
+        VERSION
+        internal
```

Comment:
this could be dangerous if we have a downstream patch that isn't yet available in the upstream release? It's unlikely to happen since we should only cherry-pick commits that are merged upstream but I can imagine a patch that is only in the main master, we backport to our v2. downstream x.0 and later on, upstream releases v2.x.1 (without the said patch).